packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx

```diff
@@ -76,9 +87,28 @@ const MessageBubble = (props: ChatMessageComponentAsMessageBubbleProps): JSX.Ele
     onRenderAvatar,
     showMessageStatus,
     messageStatus,
-    fileDownloadHandler
+    fileDownloadHandler,
+    /* @conditional-compile-remove(dateTimeCustomization) */
+    onDisplayDateTimeString
   } = props;
 
+  // onDisplayDateTimeString overwrites onDisplayDateTimeString overwrites default
+  let formattedTimestamp: string | undefined = undefined;
+  if (message.createdOn) {
+    // default
+    formattedTimestamp = showDate
+      ? formatTimestampForChatMessage(message.createdOn, new Date(), strings)
+      : formatTimeForChatMessage(message.createdOn);
+
+    /* @conditional-compile-remove(dateTimeCustomization) */
+    if (onDisplayDateTimeString) {
```

Comment:
Should we use onDisplayDateTimeString if showDate is false?

Reply:
when ShowDate is false, timestamps are still displayed but displayed without dates, only time. So show date is a way to customize default date time format. onDisplayDateTimeString should overwrite showdate